documentation/docs/25-build-and-deploy/90-adapter-vercel.md

```diff
@@ -153,6 +153,14 @@ export function load() {
 
 Since all of these variables are unchanged between build time and run time when building on Vercel, we recommend using `$env/static/private` — which will statically replace the variables, enabling optimisations like dead code elimination — rather than `$env/dynamic/private`.
 
+## Skew protection
```

Comment:
Does SvelteKit have a notion of static site generation where the static page responses should also set the cookie?

Reply:
No, prerendered pages bypass the server altogether — I guess we would need to use middleware if we wanted to protect those too

Reply:
There should be a BOA mechanism to put this into the produced resource rather than a route

Reply:
I'm not aware of such a mechanism, what should I be looking for?

Reply:
Something like "deployment path metadata". Not 100% sure

Reply:
@styfle figured out how to fix this — we set a wildcard route that adds the header for all document requests. much simpler!